cmd/gardenlet/app/migration.go

```diff
@@ -35,6 +43,43 @@ func (g *garden) runMigrations(ctx context.Context, log logr.Logger, _ cluster.C
 	return deleteOrphanedValiVPAs(ctx, log, g.mgr.GetClient())
 }
 
+// TODO(timuthy): Remove after v1.96 was released.
+// updateGardenerResourceManager updates all GRM deployments in a seed cluster,
+// in order to roll out the Brotli compression support added in Gardener v1.96.
+func updateGardenerResourceManager(ctx context.Context, log logr.Logger, cl client.Client) error {
+	image, err := imagevector.ImageVector().FindImage(imagevector.ImageNameGardenerResourceManager)
+	if err != nil {
+		return err
+	}
+	image.WithOptionalTag(version.Get().GitVersion)
+
+	grmDeployments := &appsv1.DeploymentList{}
+	if err := cl.List(ctx, grmDeployments, client.MatchingLabels(map[string]string{"app": "gardener-resource-manager"})); err != nil {
+		return err
+	}
+
+	fns := make([]flow.TaskFn, 0, len(grmDeployments.Items))
+	for _, deployment := range grmDeployments.Items {
+		grmDeployment := deployment
```

Comment:
Is this line still needed now that we have Go 1.22?

Reply:
No it shouldn't be needed, but chances were that it should be this way due to consistency 😄